### ISSUE TITLE
Unable to work with files or paths with spaces included in the name

### ISSUE THREAD

**Issue by Garulf** [deuteronomy-works/pyffmpeg#413]
code: 
```
ff = FFmpeg()
input = "video 2.mp4"
output = "video 2.jpg"
ff.options(f'-i "{input}" -ss 00:00:01.000 -vframes 1 -f image2 "{output}"')
```

error:
```
ffmpeg version 2021-12-02-git-4a6aece703-full_build-www.gyan.dev Copyright (c) 2000-2021 the FFmpeg developers
  built with gcc 11.2.0 (Rev2, Built by MSYS2 project)
  configuration: --enable-gpl --enable-version3 --enable-static --disable-w32threads --disable-autodetect --enable-fontconfig --enable-iconv --enable-gnutls --enable-libxml2 --enable-gmp --enable-lzma --enable-libsnappy --enable-zlib --enable-librist --enable-libsrt --enable-libssh --enable-libzmq --enable-avisynth --enable-libbluray --enable-libcaca --enable-sdl2 --enable-libdav1d --enable-libdavs2 --enable-libuavs3d --enable-libzvbi --enable-librav1e --enable-libsvtav1 --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxavs2 --enable-libxvid --enable-libaom --enable-libopenjpeg --enable-libvpx --enable-libass --enable-frei0r --enable-libfreetype --enable-libfribidi --enable-libvidstab --enable-libvmaf --enable-libzimg --enable-amf --enable-cuda-llvm --enable-cuvid --enable-ffnvcodec --enable-nvdec --enable-nvenc --enable-d3d11va --enable-dxva2 --enable-libmfx --enable-libshaderc --enable-vulkan --enable-libplacebo --enable-opencl --enable-libcdio --enable-libgme --enable-libmodplug --enable-libopenmpt --enable-libopencore-amrwb --enable-libmp3lame --enable-libshine --enable-libtheora --enable-libtwolame --enable-libvo-amrwbenc --enable-libilbc --enable-libgsm --enable-libopencore-amrnb --enable-libopus --enable-libspeex --enable-libvorbis --enable-ladspa --enable-libbs2b --enable-libflite --enable-libmysofa --enable-librubberband --enable-libsoxr --enable-chromaprint
  libavutil      57.  9.101 / 57.  9.101
  libavcodec     59. 14.100 / 59. 14.100
  libavformat    59.  9.102 / 59.  9.102
  libavdevice    59.  0.101 / 59.  0.101
  libavfilter     8. 18.100 /  8. 18.100
  libswscale      6.  1.101 /  6.  1.101
  libswresample   4.  0.100 /  4.  0.100
  libpostproc    56.  0.100 / 56.  0.100
"video: Invalid argument
```

It seems like this line is causing the issue:

https://github.com/deuteronomy-works/pyffmpeg/blob/d1b67862ea1daafd33f64206a1657505584f2b50/pyffmpeg/__init__.py#L178

By extracting the `_ffmpeg_instance['options'].args` property you can see how mangled the file names get"
```
['C:\\Users\\Garulf\\.pyffmpeg\\bin\\ffmpeg.exe', '-y', '-i', '"static\\media\\video', '2.mp4"', 
'-ss', '00:00:01.000', '-vframes', '1', '-f', 'image2', '"./static/video_thumbnails/video', '2.jpg"']
```

**Comment by amoh-godwin** [comment]
You are right @Garulf, working on it

**Comment by amoh-godwin** [comment]
@Garulf a new version has been released; version 2.3.0.3.
Thank you for reporting the problem
